test/verify/check-packagekit

```diff
@@ -54,9 +54,15 @@ class TestUpdates(MachineCase):
         m.start_cockpit()
         b.login_and_go("/updates")
 
+        # PK starts from a blank state, thus should force refresh
+        b.wait_in_text("#state", "Refreshing")
```

Comment:
This seems prone to races if the test machines are slow or overloaded. In theory at least.